src/a.tex

```diff
@@ -428,14 +428,28 @@ \section{Atomic Memory Operations}
 
 \vspace{-0.1in} The atomic memory operation (AMO) instructions perform
 read-modify-write operations for multiprocessor synchronization and
-are encoded with an R-type instruction format.  These AMO instructions
-atomically load a data value from the address in {\em rs1}, place the
-value into register {\em rd}, apply a binary operator to the loaded
-value and the original value in {\em rs2}, then store the result back
-to the original address in {\em rs1}. AMOs can either operate on 64-bit (RV64
-only) or 32-bit words in memory.  For RV64, 32-bit AMOs always
-sign-extend the value placed in {\em rd}, and ignore the upper 32 bits
-of the original value of {\em rs2}.
+are encoded with an R-type instruction format. A data value is atomically
+loaded from the address in {\em rs1} and placed into {\em rd}, then a
+binary operator is applied to the loaded value and the original value in
+{\em rs2}, finally the result is stored back to the original address.
```

Comment:
I can't see how this rewording clarifies anything.

Reply:
Point taken, will drop this change